controlplane/internal/controller/agentcontrolplane_controller.go

```diff
@@ -85,20 +87,33 @@ type AgentControlPlaneReconciler struct {
 func (r *AgentControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, rerr error) {
 	log := ctrl.LoggerFrom(ctx)
 
-	// handle deletion
 	acp := &controlplanev1beta1.AgentControlPlane{}
 	if err := r.Client.Get(ctx, req.NamespacedName, acp); err != nil {
 		if apierrors.IsNotFound(err) {
 			return ctrl.Result{}, nil
 		}
 		return ctrl.Result{}, err
 	}
+	log.WithValues("AgentControlPlane Name", acp.Name, "AgentControlPlane Namespace", acp.Namespace)
+	log.V(5).Info("Started reconciling AgentControlPlane")
 	defer func() {
 		if rerr = r.Client.Status().Update(ctx, acp); rerr != nil {
-			log.Error(rerr, "couldn't update AgentControlPlane Status", "name", acp.Name, "namespace", acp.Namespace)
+			log.V(3).Error(rerr, "couldn't update AgentControlPlane Status")
```

Comment:
Shouldn't this be V(3) info? It seems strange to use verbosity for error

Reply:
Ah yeah sure, I'll change it back to error - wasn't sure if it was that important to see all the time

Reply:
Changed it!